test/app.test.js

```diff
@@ -0,0 +1,12 @@
+const request = require('supertest');
+const { createApp } = require('../app');
+
+describe('health check', () => {
+  beforeAll(async () => {
+    app = createApp();
+  });
+
+  test('SUCCESS : health check', async () => {
+    await request(app).get('/ping').expect(200).expect({ message: 'pong' });
+  });
```

Comment:
```suggestion
  test('SUCCESS : health check', async () => {
    const response = request(app_.get(/ping)
    
    expect(request.statusCode).toEqual(200)
    expect(request.body).toEqual({ message: 'pong' })
  });
```
잘 작성해주셨네요 은경님!! 위와 같이 response에 응답을 담고 jest expect matcher를 활용하면 더 유연한 테스트 코드를 작성할 수 있습니다.

Reply:
종범님 혹쉬... 저 공부하라고... 헷갈리게 적어주신거죠..?

Reply:
헷갈리게 해서 죄송해요🤣

Reply:
아니에욬ㅋㅋㅋ